recipes/cmdstan/build.sh

```diff
@@ -1,11 +1,14 @@
+mkdir -p $PREFIX/bin
+cp -r . $PREFIX/bin/cmdstan
+
+cd $PREFIX/bin/cmdstan
+
 echo "TBB_CXX_TYPE=$c_compiler"  >> make/local
 echo "TBB_INTERFACE_NEW=true" >> make/local
 echo "TBB_INC=$PREFIX/include/" >> make/local
 echo "TBB_LIB=$PREFIX/lib/" >> make/local
 
 make clean-all
 
-make build -j4
+make build -j${CPU_COUNT}
 
```

Comment:
there is no `make install` btw?

Reply:
Missed this comment -- no, not currently